### PR TITLE
Merge in "get it working" changes from master

### PR DESCRIPTION
**Pull request by Hoolean**

### CODE REVIEW

File: src/OagBundle/Resources/views/Default/index.html.twig [hunk 1]
```diff
@@ -54,7 +54,7 @@
         </div>
     </ul>
 
-    <h2>Enhancement Documents</h2>
+    <h2>Enhancement Documents <small><small>[activities/locations]</small></small></h2>
```
Comment (ob6160) [inline src/OagBundle/Resources/views/Default/index.html.twig]: Why are we nesting two small tags here?

File: src/OagBundle/Resources/public/css/main.css [hunk 2]
```diff
@@ -78,3 +78,14 @@ body {
 .leaflet-tile.leaflet-tile-loaded {
     margin-top: 0;
 }
+
+.biglabel {
+    font-size: 105%;
+    font-weight: bolder;
```
Comment (ob6160) [inline src/OagBundle/Resources/public/css/main.css]: Not heard of `bolder` before, is it even more bold than **bold**? :grin:

File: app/config/config_test.yml [hunk 3]
```diff
@@ -20,7 +20,7 @@ doctrine:
         charset: UTF8
         driver: pdo_sqlite
         path: '%kernel.project_dir%/app/test.db'
-
+
```
Comment (ob6160) [inline app/config/config_test.yml]: Spaces!

File: src/OagBundle/Resources/views/Default/index.html.twig (hunk 1 above)
Comment (Hoolean) [reply src/OagBundle/Resources/views/Default/index.html.twig]: To go even smaller ;)

image

Comment (ob6160) [reply src/OagBundle/Resources/views/Default/index.html.twig]: Whoa

**Pull request merged**